themes/custom/az_barrio/templates/blocks/block--az-barrio-sidebar-menu.html.twig

```diff
@@ -46,7 +46,7 @@
     {% set title_attributes = title_attributes.addClass('sr-only') %}
   {% endif %}
   {{ title_prefix }}
-  <h2{{ title_attributes.setAttribute('id', heading_id).addClass('h4').addClass('mt-md-5') }}>{{ configuration.label }}</h2>
+  <h2{{ title_attributes.setAttribute('id', heading_id).addClass('h4').addClass('mt-0') }}>{{ configuration.label }}</h2>
```

Comment:
This change immediately affects existing sites. Please change to only affect new sites.

Reply:
I'll supply a patch so we can test this in combination with the #1834 so we can test this easily on existing sites and figure out a way to do the least amount of harm.

I can't think of a way to only affect existing sites, without rewriting this template a bit first.
Steps would be:
- Move the added classes into a theme setting. `mt-0`
- Add an update that changes the theme setting back to what it was before this change. `mt-md-5`

Updates only run on existing sites. We would be banking on the fact that most sites have changed theme settings ,making config distro skip updating the theme settings. (we'd need to test that this is true, not sure if it is our not).

Reply:
Additionally, we can update this setting via drush on any sites we deem safe, after this.

Reply:
Of course this will most likely be a problem for any sites that have a sub theme with  a different template and those added classes changed.